bliss/api.py

```diff
@@ -133,14 +133,14 @@ def train_on_cached_data(
         _train(cfg)
 
     def load_survey(self, survey: SurveyType, run, camcol, field, download_dir: str):
-        SDSSDownloader(run, camcol, field, self.cwd + f"/{download_dir}").download_all()
+        SDSSDownloader([(run, camcol, field)], self.cwd + f"/{download_dir}").download_all()
         # assert files downloaded at download_dir
 
     def predict_sdss(
         self,
         weight_save_path: str,
         **kwargs,
-    ) -> Tuple[FullCatalog, Table, Table]:
+    ) -> Tuple[FullCatalog, Table, Dict[Any, Table]]:
```

Comment:
Is `image_id` always an integer? If so, can we get rid of `Any` here?

Reply:
Not always an integer. For example, for SDSS it is `Tuple[int, int, int]` for `(run, camcol, field)`; for DECaLS it is `str` for `brickname`.